elide-core/src/main/java/com/yahoo/elide/core/filter/dialect/RSQLFilterDialect.java

```diff
@@ -313,11 +321,29 @@ public FilterExpression visit(ComparisonNode node, Class entityType) {
                 } else {
                     return new NotFilterExpression(new Predicate(path, Operator.IN, values));
                 }
+            } else if (op == isnullOp) {
```

Comment:
It is too late to put this here as we already have attempted to coerce the values above.